js/woo.js

```diff
@@ -710,7 +710,10 @@ module.exports = class woo extends Exchange {
                         if (price === undefined) {
                             throw new InvalidOrder (this.id + " createOrder() requires the price argument for market buy orders to calculate total order cost. Supply a price argument to createOrder() call if you want the cost to be calculated for you from price and amount, or alternatively, supply the total cost value in the 'order_amount' in  exchange-specific parameters");
                         } else {
-                            request['order_amount'] = this.costToPrecision (symbol, amount * price);
+                            const amountString = amount.toString ();
+                            const priceString = price.toString ();
+                            const orderAmount = Precise.stringMul (amountString, priceString);
+                            request['order_amount'] = this.costToPrecision (symbol, orderAmount);
```

Comment:
We can't use .toString () here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

<img width="1515" alt="Screenshot 2022-09-06 at 03 57 27" src="https://user-images.githubusercontent.com/1294454/188525527-ac94d960-077b-4b6b-a478-4fb97e706a5e.png">

Let's use `numberToString` instead.

Reply:
update

```
% woo createOrder ADA/USDT market buy 3 0.45
2022-09-06T02:38:38.126Z
Node.js: v18.4.0
CCXT v1.93.3
(node:4368) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
woo.createOrder (ADA/USDT, market, buy, 3, 0.45)
2022-09-06T02:38:39.521Z iteration 0 passed in 359 ms

{
  id: '162243910',
  clientOrderId: '0',
  timestamp: 1662431919602,
  datetime: '2022-09-06T02:38:39.602Z',
  lastTradeTimestamp: undefined,
  status: undefined,
  symbol: 'ADA/USDT',
  type: 'market',
  timeInForce: 'IOC',
  postOnly: undefined,
  reduceOnly: undefined,
  side: undefined,
  price: undefined,
  stopPrice: undefined,
  average: undefined,
  amount: undefined,
  filled: undefined,
  remaining: undefined,
  cost: 1.35,
  trades: [],
  fee: { cost: undefined, currency: undefined },
  info: {
    success: true,
    timestamp: '1662431919.602',
    order_id: '162243910',
    order_type: 'MARKET',
    order_price: null,
    order_quantity: null,
    order_amount: '1.35',
    client_order_id: '0'
  },
  fees: [ { cost: undefined, currency: undefined } ]
}
2022-09-06T02:38:39.521Z iteration 1 passed in 359 ms
```